packages/ciphernode/net/src/retry.rs

```diff
@@ -1,70 +0,0 @@
-use anyhow::Result;
-use std::{future::Future, time::Duration};
-use tokio::time::sleep;
-use tracing::{error, warn};
-
-pub enum RetryError {
-    Failure(anyhow::Error),
-    Retry(anyhow::Error),
-}
-
-pub fn to_retry(e: impl Into<anyhow::Error>) -> RetryError {
-    RetryError::Retry(e.into())
-}
-
-pub const BACKOFF_DELAY: u64 = 500;
-pub const BACKOFF_MAX_RETRIES: u32 = 10;
-
-/// Retries an async operation with exponential backoff
-///
-/// # Arguments
-/// * `operation` - Async function to retry
-/// * `max_attempts` - Maximum number of retry attempts
-/// * `initial_delay_ms` - Initial delay between retries in milliseconds
-///
-/// # Returns
-/// * `Result<()>` - Ok if the operation succeeded, Err if all retries failed
-pub async fn retry_with_backoff<F, Fut>(
```

Comment:
I think this is going to be really useful as a function as we go to harden up our actors. Eg. ensuring our set and forget  database writes actually happen so I think it is better to attempt to use this function as opposed to bake it in to the dialer.